README.md

```diff
@@ -23,8 +23,7 @@
 
 # 📖 [To-Do-List Project] <a name="about-project"></a>
 
-**[To-Do-List Project]** This project build with a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this repo. This simple web page will be built using webpack and served by a webpack dev server.
-
+**[To-Do-List Project]** In this project, I implement the CRUD (create, read, update, delete) methods. All the elements of the user interface will be fully functional.
 ## 🛠 Built With <a name="built-with"></a>
 
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove redundant demo links in your ReadMe, on _line 58_.

Reply:
well resolved! ✔️